client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -14,8 +14,9 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 	return (
 		<li
 			className={ classnames( 'launchpad__task', {
-				'keep-active': keepActive,
-				'is-completed': isCompleted,
+				'keep-active': keepActive && isCompleted, // a task that is completed and can be revisited
```

Comment:
Can we switch these conditions around for the sake of readability since the other checks are for `isCompleted` _and then_ `keepActive`?